src/app/common/paging/pageable-table/pageable-table.component.ts

```diff
@@ -14,8 +14,9 @@ export class PageableTableComponent {
 	@Input() loading = false;
 	@Input() showInCard = false;
 	@Input() showActions = false;
+	@Input() hideActionsNoData = true;
```

Comment:
I think we should set the default to `false`.  That will make it's use consistent with `hideTableNoData`.  I think our 2 downstream projects are the only ones we have to worry about for compatibility and I am not actually using table actions anywhere currently.

Reply:
Great! It is done.